build.gradle

```diff
@@ -19,13 +19,15 @@ apply from: "$rootDir/gradle-scripts/extensions.gradle"
 apply from: "$rootDir/gradle-scripts/project-info.gradle"
 apply from: "$rootDir/gradle-scripts/java-compile.gradle"
 apply from: "$rootDir/gradle-scripts/repositories.gradle"
-apply from: "$rootDir/gradle-scripts/integration-tests.gradle"
-apply from: "$rootDir/gradle-scripts/test-logging.gradle"
 apply from: "$rootDir/gradle-scripts/dependencies.gradle"
+
 apply from: "$rootDir/gradle-scripts/checkstyle.gradle"
 apply from: "$rootDir/gradle-scripts/pmd.gradle"
-apply from: "$rootDir/gradle-scripts/jacoco.gradle"
 apply from: "$rootDir/gradle-scripts/findbugs.gradle"
+
+apply from: "$rootDir/gradle-scripts/integration-tests.gradle"
+apply from: "$rootDir/gradle-scripts/test-logging.gradle"
+apply from: "$rootDir/gradle-scripts/jacoco.gradle"
```

Comment:
Changing the order of applying of Gradle scripts doesn't change the actual order of execution. 
I have also tried running it just in case, and it indeed didn't change the order of execution of Gradle tasks. Have you tried building the project after your change?

This article could be of use to you about order of execution of Gradle tasks: http://trickyandroid.com/gradle-tip-3-tasks-ordering/